app/boards/shields/zmk_uno/zmk_uno.zmk.yml

```diff
@@ -0,0 +1,8 @@
+file_format: "1"
```

Comment:
Missed changing this file

Reply:
Fixed, and added the `arduino_uno` interconnect details.